### ISSUE TITLE
Group header can not use

### ISSUE THREAD

**Issue by gongxh13** [akbarpulatov/flutter_awesome_select#18]
When set `groupEnabled: true,`， throw below exception:
```
The following _CastError was thrown building S2GroupHeader(dirty):
type 'List<Widget?>' is not a subtype of type 'List<Widget>' in type cast

The relevant error-causing widget was: 
  GetMaterialApp GetMaterialApp:file:///C:/Users/dell/Desktop/gongxhfile/reliser/lechat/lib/main.dart:40:22
When the exception was thrown, this was the stack: 
#0      S2GroupHeader.build (package:awesome_select/src/group_header.dart:37:43)
```
The error code is in group_header.dart:
![image](https://user-images.githubusercontent.com/44743148/141413739-d509d6bb-0a15-4e63-b3bd-3a48fab5a927.png)

**Comment by absar** [comment]
It's duplicate of https://github.com/akbarpulatov/flutter_awesome_select/issues/13

**Comment by vasilich6107** [comment]
I released https://pub.dev/packages/flutter_awesome_select with fixed null safety.
Could you check that everything works for you?

**Comment by akbarpulatov** [comment]
Issue is fixed on 834321de79dbb2b61befcfa8ddd97c90d2569c55
